pyccel/codegen/printing/pycode.py

```diff
@@ -41,6 +41,8 @@ def _construct_header(func_name, args):
                    'empty_like' : 'empty',
                    'zeros_like' : 'zeros',
                    'ones_like'  : 'ones',
+                   'amax'        : 'max',
+                   'amin'        : 'min',
```

Comment:
This should be `'min' : 'amin', 'max': 'amax'`, not `'amin' : 'min', 'amax': 'max'`. The value returned by the dictionary is the one printed